assets/_scss/core_design.scss

```diff
@@ -1,3 +1,7 @@
+html {
+    scroll-padding-top: 7rem;
```

Comment:
Nemělo by to být konzistentní s body { padding-top}, tedy 56px? Stejně tak bych čekal, že se to z js aktualizuje pro dev-warning (podobně jako assets/js/main.js:6)

Reply:
7rem je dostačující pro obě verze, takže za mě je to takhle prozatím v pohodě. (Ideálně bych tyhle a související styly ještě trochu překopal, ale zatím fungují dobře.)